node/src/truv.js

```diff
@@ -2,6 +2,13 @@ import fetch from "node-fetch";
 
 const { API_CLIENT_ID, API_SECRET, API_PRODUCT_TYPE } = process.env;
 
+if (!API_CLIENT_ID || !API_SECRET) {
```

Comment:
Also add check for API_PRODUCT_TYPE here

Reply:
And don't forget to copy this change to other languages

Reply:
> Also add check for API_PRODUCT_TYPE here

it's already checked here: https://github.com/truvhq/quickstart/blob/master/node/src/serve.js

Reply:
> And don't forget to copy this change to other languages

I checked them all, it seems ok, but except c-sharp...
Unfortunately, this lang is not my cup of tea. :(